crates/rspack_plugin_javascript/src/visitors/dependency/util.rs

```diff
@@ -206,3 +206,12 @@ fn test() {
     type_args: None
   }));
 }
+
+pub fn is_unresolved_require_member_expr(expr: &Expr, unresolved_ctxt: &SyntaxContext) -> bool {
```

Comment:
Would you change the name of this function since it can be used for other expressions too?

Reply:
Something that is not related to `require` looks fine to me, maybe something like `is_unresolved_member_object_ident`

Reply:
Would you mind me pushing some code to this PR?